rskj-core/src/main/java/co/rsk/RskContext.java

```diff
@@ -1264,7 +1264,9 @@ private StaticMessages getStaticMessages() {
     private MinerClock getMinerClock() {
         if (minerClock == null) {
             minerClock = new MinerClock(
-                    getRskSystemProperties().getBlockchainConfig() instanceof RegTestConfig, Clock.systemUTC()
+                    getRskSystemProperties().getBlockchainConfig() instanceof RegTestConfig,
```

Comment:
In other PR we should address this so this configuration doesn't depends directly on NetworkType. Instead it should a be a configurable property on some network (not block) dependandt configuration, like ChainID